content/reference/plugins/jenkins/index.md

```diff
@@ -47,15 +47,15 @@ You can use the Pipeline Snippet Generator to help you use the Jenkins Plugin. C
 
 {{< img src="pipeline-generator.png" alt="Snippet Generator" >}}
 
-You can specify the id of an API Token stored in a secret text credential if you don't want to use the one configured globally. Choose one of the simulation in the drop-down, then click Generate Groovy. Copy and paste the result in your Pipeline script, eg:
+You can specify the id of an API Token stored in a secret text credential if you don't want to use the one configured globally. Choose one of the simulations in the drop-down menu, then click Generate Groovy. Copy and paste the result in your Pipeline script, eg:
 ```groovy
 // Declarative Pipeline Syntax
 pipeline {
     agent any
     stages {
         stage("Gatling Enterprise simulation") {
             steps {
-                gatlingFrontLineLauncherStep credentialId: '6737158c-0ff6-4033-91ad-6f3a811aab52', simulationId: '00eacd1c-ef91-4076-ad57-99b4c6675a9e'
+                gatlingFrontLineLauncherStep credentialId: 'GATLING_API_TOKEN', simulationId: '00eacd1c-ef91-4076-ad57-99b4c6675a9e'
```

Comment:
Why replace GATLING_API_TOKEN and not the smulationId?

Reply:
Because of the format - users can set their own keys for the credentials in Jenkins, so it feels very weird to me to assume they'll use an UUID.

Reply:
ok